lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -587,11 +587,13 @@ namespace {
       return result;
     }
 
+    LLVM_ATTRIBUTE_UNUSED // If unused, silence -Wunused-function
```

Comment:
This whole class definition is dead actually. See https://github.com/apple/swift/pull/27535. Do you want to incorporate my change into your PR?

Reply:
Are you tired of begging CI to try again? :-P

But sure, ya, these two PRs will conflict. I'll update this PR.